Seminars/Sem.01/Pract.01/task04.cpp

```diff
@@ -0,0 +1,30 @@
+<iostream>
+
+using namespace std;
+
+struct Student {
+   char name[30];
+   int facultyNumber;
+};
+
+void initStudent(Student &student) {
+   cout << "Enter name: ";
+   cin.getline(student.name, 30);
+   do {
+       cout << "Enter faculty number: ";
+       cin >> student.facultyNumber;
+   } while (student.facultyNumber <= 0);
+   cin.ignore();
+}
+
+void printStudent(Student const &student) {
```

Comment:
Would prefer const student, rather than student constant, but both are viable if you prefer it this way.